lib/widgets/theme.dart

```diff
@@ -327,6 +330,7 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
   final Color modalBarrierColor;
   final Color mutedUnreadBadge;
   final Color navigationButtonBg;
+  final Color pressedTint;
```

Comment:
Bump on https://github.com/zulip/zulip-flutter/pull/1152#discussion_r1919332516 : this doesn't belong in this section, which has this description:

```dart
  // Not named variables in Figma; taken from older Figma drafts, or elsewhere.
```

This _is_ a named variable in Figma, as it seems you've noticed 🙂

https://github.com/zulip/zulip-flutter/pull/1152#discussion_r1920456371
> It is named as `pressedTint` itself in the design file.

so it goes in the same section as other named Figma variables, so (in current `main`) between `mainBackground` and `textInput`.